libs/coin-evm/src/api/transaction/index.ts

```diff
@@ -0,0 +1,15 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { getExplorerApi } from "../explorer";
+
+// TODO: implement getTransactionByHash for etherscan explorer
```

Comment:
can't be done in this PR ?

Reply:
Unecessary, a transaction can be fetched from a node with the rpc method `eth_gettransactionbyhash` which is provided by `ethers` with the `provider.getTransaction( hash )` helper

Reply:
It's also provided by the `node` api with [this](https://github.com/LedgerHQ/ledger-live/blob/bbad6146cee4fa55a8d0e2fec47579c945e6e59b/libs/coin-evm/src/api/node/types.ts#L7)

Reply:
So should we delete this TODO or add a ticket in the backlog ?

Reply:
Turns out this whole `getTransactionByHash` logic is unnecessary and a duplication of the `getTransaction` function available through the node API as mentioned by @lambertkevin 

fixed here 9320e023af14319ab3ad4a6ef350426aa5f6d195